src/lumigo_opentelemetry/__init__.py

```diff
@@ -184,7 +210,35 @@ def init() -> Dict[str, Any]:
 
     trace.set_tracer_provider(tracer_provider)
 
-    return {"tracer_provider": tracer_provider}
+    if logging_enabled:
+        _logs.set_logger_provider(logger_provider)
+
+        # Add the handler to the root logger, hence affecting all loggers created by the app from now on
+        logging.getLogger().addHandler(LoggingHandler(logger_provider=logger_provider))
+
+        # Inject the span context into logs
+        LoggingInstrumentor().instrument(set_logging_format=True)
+
+        if logdump_file:
+            from opentelemetry.sdk._logs.export import (
+                ConsoleLogExporter,
+                SimpleLogRecordProcessor,
+            )
+
+            logger_provider.add_log_record_processor(
+                SimpleLogRecordProcessor(
+                    ConsoleLogExporter(
+                        out=open(logdump_file, "w"),
```

Comment:
I wonder what errors are thrown when the path is invalid

Reply:
It crashes the entire distro :/ with the file error: 
```sh
Traceback (most recent call last):
  File "/Users/harelmoshe/.pyenv/versions/3.9.9/lib/python3.9/site.py", line 589, in <module>
    main()
  File "/Users/harelmoshe/.pyenv/versions/3.9.9/lib/python3.9/site.py", line 582, in main
    execsitecustomize()
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/autowrapt/bootstrap.py", line 32, in _execsitecustomize
    return wrapped(*args, **kwargs)
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/autowrapt/bootstrap.py", line 38, in _execsitecustomize
    _register_bootstrap_functions()
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/autowrapt/bootstrap.py", line 27, in _register_bootstrap_functions
    discover_post_import_hooks(name)
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/wrapt/importer.py", line 100, in discover_post_import_hooks
    register_post_import_hook(callback, entrypoint.name)
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/wrapt/importer.py", line 79, in register_post_import_hook
    hook(module)
  File "/Users/harelmoshe/source/opentelemetry-python-distro/venv/lib/python3.9/site-packages/wrapt/importer.py", line 85, in import_hook
    __import__(entrypoint.module_name)
  File "/Users/harelmoshe/source/opentelemetry-python-distro/src/lumigo_opentelemetry/__init__.py", line 262, in <module>
    init_data = init()
  File "/Users/harelmoshe/source/opentelemetry-python-distro/src/lumigo_opentelemetry/__init__.py", line 228, in init
```
I think we better catch that and default to dumping the logs to `console:error`, WDYT?

Reply:
Added a fallback to `stdout`